packages/instrumentation-openai/src/image-wrappers.ts

```
@@ -0,0 +1,546 @@
+    if (value !== undefined) {
+      span.setAttribute(key, value);
+    }
+  });
+}
+
+export function wrapImageGeneration(
+  tracer: ReturnType<typeof trace.getTracer>,
+  uploadCallback?: ImageUploadCallback,
+  instrumentationConfig?: { enrichTokens?: boolean }
+) {
+  return function (original: Function) {
```

ESLint

[error] The `Function` type accepts any function-like value.
Prefer explicitly defining any function parameters and return type.

(@typescript-eslint/no-unsafe-function-type)